api/api.proto

```diff
@@ -128,12 +128,23 @@ message RevokeRefreshReq {
   string client_id = 2;
 }
 
-// RevokeRefreshResp determines if the refresh token is revoked successfully. 
+// RevokeRefreshResp determines if the refresh token is revoked successfully.
 message RevokeRefreshResp {
   // Set to true is refresh token was not found and token could not be revoked.
   bool not_found = 1;
 }
 
+// GetPasswordReq is a request get a specific password by email.
+message GetPasswordReq {
+  string email = 1;
+}
+
+// GetPasswordResp a specific password by email.
```

Comment:
This is really oddly named. Can we either rename it "GetUserResp" and "GetUserReq"? Or can we strongly clarify that the password isn't returned?